### PR TITLE
feature: update coloring

### PR DESCRIPTION
**Pull request by amanPras**
## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Add coloring for condition, branch, callback, serialization, subroutine and exit

## Checklist:
- [x] Each of my commits contains one meaningful change
- [x] I have performed rebase of my branch on top of the development
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by VitGottwald** [comment]
WHEN-COMPILED is a function, unrelated to EVALUATE.

**Review by amanPras — COMMENTED** [review]
feature: update coloring
Signed-off-by: Aman Prashant <aman.prashant@broadcom.com>

**Comment by amanPras** [comment]
> WHEN-COMPILED is a function, unrelated to EVALUATE.

updated

**Review by ishche — APPROVED** [review]

**Review by Nurkambay — APPROVED** [review]

File: clients/cobol-lsp-vscode-extension/syntaxes/COBOL.tmLanguage.json [hunk 1]
```diff
@@ -89,7 +80,117 @@
...
+      "patterns": [
+        {"include": "#cobol-condition-keyword"},
+        {"include": "#cobol-branch-keyword"},
+        {"include": "#cobol-callback-keyword"},
+        {"include": "#cobol-serialization-keyword"},
+        {"include": "#cobol-subroutine-keyword"},
+        {"include": "#cobol-exit-keyword"},
+        {"include": "#cobol-general-keyword"},
+        {
+          "match": "(?<![\\-\\w])(?i:END-EXEC)(?![\\-\\w])",
```
Comment (VitGottwald) [inline clients/cobol-lsp-vscode-extension/syntaxes/COBOL.tmLanguage.json]: This should be covered by the `end` block of `exec-cics-block` and `exec-sql-block`.

Comment (amanPras) [reply clients/cobol-lsp-vscode-extension/syntaxes/COBOL.tmLanguage.json]: updated

**Review by VitGottwald — APPROVED** [review]
Nice, thank you!

**Pull request merged**